src/Infrastructure.VS/Infrastructure.VS.csproj

```diff
@@ -15,7 +15,7 @@
 
   <ItemGroup Label="Package references">
     <PackageReference Include="Microsoft.VisualStudio.Setup.Configuration.Interop" Version="1.16.30" />
-    <PackageReference Include="Microsoft.VisualStudio.SDK.EmbedInteropTypes" Version="15.0.34" />
+    <PackageReference Include="Microsoft.VisualStudio.SDK.EmbedInteropTypes" Version="15.0.27" />
```

Comment:
There was a conflict with `Integration` and `Integration.Vsix`'s dependencies. I decided to downgrade the package instead of upgrading the others.